DAO.sol

```diff
@@ -782,7 +782,7 @@ contract DAO is DAOInterface, Token, TokenSale {
 
 
     function changeProposalDeposit(uint _proposalDeposit) noEther external {
-        if (msg.sender != address(this) || _proposalDeposit > actualBalance() / 10)
+        if (msg.sender != address(this) || _proposalDeposit > (actualBalance() + rewardToken[address(this)]) / 100)
```

Comment:
Maybe make this `100` into a constant and put it in the contract beginning too?
Something like:
```js
    // Denotes the maximim proposal deposit that can be given. It is given as 
    // a fraction of total Ether spent plus balance of the DAO
    uint constant maxDepositDivisor = 100;
```